lib/internal/modules/esm/translators.js

```diff
@@ -282,10 +282,15 @@ translators.set('builtin', async function builtinStrategy(url) {
   debug(`Translating BuiltinModule ${url}`);
   // Slice 'node:' scheme
   const id = StringPrototypeSlice(url, 5);
+
+  // compileForPublicLoader() throws if mod.canBeRequiredByUsers is false:
   const module = loadNativeModule(id, url, true);
```

Comment:
Weird, so this last argument was always unnecessary! Surely we should remove the `true` here because we're separately compiling and checking `canBeRequiredByUsers` below?

Reply:
It was originally removed in <https://github.com/nodejs/node/commit/796f3d0af49164314868c4778af90eca356f1fef>.